app/pods/publications/publication/publication-activities/index/route.js

```diff
@@ -0,0 +1,37 @@
+import Route from '@ember/routing/route';
```

Comment:
Ik heb gezien dat een aparte route in de "Vertalingen"-pull-request werd gesuggereerd. Ik vind dit een verwarrende aanpak. Ik zou dit liever vermijden.

Reply:
Wat vind je daar precies verwarrend aan? Door de aparte route wordt het model/logica die nodig  is voor het info-panel  gescheiden van het model/logica die nodig is voor de timeline, wat me duidelijker lijkt?

Reply:
Ik had het pattern nog nooit gezien. Ik begrijp nu wel de logica.